types/fs-extra/fs-extra-tests.ts

```diff
@@ -171,14 +171,17 @@ fs.ensureLink(path, path).then(() => {
 });
 fs.ensureLink(path, path, errorCallback);
 fs.createLink(path, path).then(() => {
-	// stub
+    // stub
 });
 fs.createLink(path, path, errorCallback);
 fs.ensureLinkSync(path, path);
 fs.createLinkSync(path, path);
 fs.ensureSymlink(path, path, "file").then(() => {
     // stub
 });
+fs.ensureSymlink(path, path, "junction").then(() => {
+	// stub
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/fs-extra/fs-extra-tests.ts;
  Line: 183;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.